quickstart-mongodb-atlas/schemas/mongodb-atlas-cluster.json

```diff
@@ -2,63 +2,125 @@
     "typeName": "MongoDB::Atlas::Cluster",
```

Comment:
Tried deploying stack with this template as is and got this error: "Template format error: At least one Resources member must be defined." can you fix/make sure it deploys with no modifications